spec/integ/crypto/verification.spec.ts

```diff
@@ -206,15 +206,21 @@ function runTests(backend: string, initCrypto: InitCrypto, methods: string[] | u
                     short_authentication_string: ["decimal", "emoji"],
                 },
             });
-            await waitForVerificationRequestChanged(request);
-            expect(request.phase).toEqual(VerificationPhase.Started);
-            expect(request.otherPartySupportsMethod("m.sas.v1")).toBe(true);
-            expect(request.chosenMethod).toEqual("m.sas.v1");
-
-            // there should now be a verifier
-            const verifier: Verifier = request.verifier!;
-            expect(verifier).toBeDefined();
-            expect(verifier.getShowSasCallbacks()).toBeNull();
+            // as soon as the Changed event arrives, `verifier` should be defined
+            const verifier = await new Promise<Verifier>((resolve) => {
+                function onChange() {
+                    expect(request.phase).toEqual(VerificationPhase.Started);
+                    expect(request.otherPartySupportsMethod("m.sas.v1")).toBe(true);
+                    expect(request.chosenMethod).toEqual("m.sas.v1");
```

Comment:
Should this be `toBe`?

Reply:
Possibly? I'm not sure it makes any difference, or if there is any particluar convention here.

Reply:
I was only asking as using `toEqual` suggests to me some sort of object comparison is going on (comparing keys and values I mean) whereas `toBe` suggests comparison of primitives. It makes no difference to the output here.

Reply:
Well, we don't normally write `method.is("m.sas.v1")` for doing string comparison, so I'm not convinced that `toBe` is, in fact, more natural for string comparison.

Also, I come from a Python world, where you definitely need `assertEqual`, and not `assertIs`:

```
>>> a = "foo"
>>> a += "bar"
>>> b = "foobar"
>>> a == b
True
>>> a is b
False
```